### ISSUE TITLE
Media Attribution headings don't follow expected heading order

### ISSUE THREAD

**Issue by SteelWagstaff** [pressbooks/pressbooks#4044]
Currently, when we render media attributions at the end of a chapter, the heading for this section is a 3rd-level heading. It should be a second level heading so that headings nest correctly and don't skip a level (from H1 to H3). H2 is the most appropriate heading level for this page component.

**Comment by cmurtagh** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Pressbooks-Product-Backlog-60ae7d807f6e370010b3d5be/issues/gh/pressbooks/pressbooks/4044?planning-poker) with Zenhub @arzola @fdalcin @ho-man-chan @MichelleWeremczuk @richard015ar @tw77

**Comment by cmurtagh** [comment]
I've already reviewed the changes, they were pretty trivial. Just needs to be tested before being released.

**Comment by richard015ar** [comment]
@SteelWagstaff, when you add a `<h2>` (heading) in the chapter and then add an image with attribution within that section, you’ll get two `h2` despite the image being inside the subtitle. Example:

![Image](https://github.com/user-attachments/assets/8ee3c412-5c25-46eb-ada4-3e7d6733d63f)

I am not sure if that's a problem or that's ok/expected. It looks like a second heading, but I am not sure if that's ok and depends on how the chapter author writes its content, how maybe we can add an `<hN>` where `N = last heading - 1`. Does it make sense? or it doesn't have nothing to do with the final purpose.

**Comment by SteelWagstaff** [comment]
Hi @richard015ar thanks for checking. Yes, that's expected behavior. In this case the page structure would look like 
`<h1>Chapter</h1>`
content
`<h2>Some heading here<h2>`
more content (could included several more headings of any level
end of chapter
`<h2>Media Attributions<h2>`

By making media attributions ALWAYS an H2, we're ensuring it will _never_ break expected heading order. Every chapter has an H1 already (the chapter title), so the appropriate heading for an element like this will always be H2. Even if that chapter has 15 other H2s, 10 H3s, and 10 H4s, media attributions should always be an H2 -- and will always be a lower level heading that the chapter title and a sibling to other H2 sections within the chapter. I think that's the way we want it. Currently, if the chapter doesn't have any H2 elements, the media attributions element will break expected sequence, because the chapter will have an H1 (the chapter title) and then it jumps to this heading (H3), skipping H2 entirely, which is not the intended result.

**Comment by richard015ar** [comment]
@cmurtagh this was merged already, and I think it was released as well. Let's @arzola confirm this.
